### ISSUE TITLE
🐛 [BUG] : Flowing github star counter in header

### ISSUE THREAD

**Issue by xuelink** [langx/website#31]
**Describe the bug**
Flowing github star counter

**Expected behavior**
It should stick to theme toggle

**Screenshots**
<img src="https://github.com/languageXchange/website/assets/3280475/80103aec-86ae-42f3-aa9b-1d77ab420fbc" alt="Simulator Screenshot" width="300">

**Smartphone (please complete the following information):**
iPhone SE

**Additional**
Related with #20 

_If you want speed up process of this issue, please send 🚀 as a reaction._

**Comment by neebooo** [comment]
Can you assign this to me please

**Comment by behics** [comment]
> Can you assign this to me please

Done! Thank you for interest.

**Comment by xuelink** [comment]
> Can you assign this to me please

We miss you @neebooo :) Please, tell me if you can't have time to to that, let me do this issue free to be assignable.

**Comment by xuelink** [comment]
I think you can fix this bug @jakebogan01 , let me know if you want to be assigned this issue as well 
Best

**Comment by jakebogan01** [comment]
@xuelink Yeah I can try this one. Let me finish the footer issue first and then I can attempt this issue next

**Comment by jakebogan01** [comment]
@xuelink I have viewed the website on all major browsers, mobile and desktop. I can't replicate this bug, nor do I even see the GitHub star in general. Is this still an issue somehow?

**Comment by xuelink** [comment]
@jakebogan01 

https://github.com/langx/website/assets/3280475/84b3bea1-d32d-4095-9fd2-c8f995c8d670

**Comment by xuelink** [comment]
You may play with CSS to make github icon and number horizontally instead of vertically, it may fix 😉

**Comment by jakebogan01** [comment]
@xuelink Thank you for the video, I was testing on the live site. I didn't know it was commented out. 

I made a PR for this issue [here](https://github.com/langx/website/pull/58)

**Comment by xuelink** [comment]
@jakebogan01 did you check with big numbers :D somethink like more than 1000, is it gonna be 1K :D we will be soon hope :)

**Comment by jakebogan01** [comment]
@xuelink Yes, I made sure that it can contain as many numbers as needed. I also made sure that a comma will be in place for example: 45,000

**Comment by xuelink** [comment]
<img width="252" alt="Screenshot 2024-04-18 at 10 19 50 PM" src="https://github.com/langx/website/assets/3280475/ab11b4cb-d35f-42ac-9fcd-b71d19dee545">

```js
https://api.github.com/repos/appwrite/appwrite?page=$i&per_page=100
```

its ok, np :) thank you!

**Comment by jakebogan01** [comment]
No problem, hope so haha

**Comment by xuelink** [comment]
I realized with links like cookies or other pages, github star counter doesnt work, but with /blog it works @jakebogan01 

https://github.com/langx/website/assets/3280475/5ef39828-2ddd-4ea4-a277-60d0be9aca39

**Comment by jakebogan01** [comment]
@xuelink that's weird. I never moved any of the api fetch logic, I only added the github icon. I did move the github count from outside the icon to inside of it but that shouldn't have broken it. I'll take a look and see what I can find out about that

**Comment by jakebogan01** [comment]
@xuelink Locally, the github stars works on all pages, blogs, privacy, terms and cookies and so on. But on the live website, cookies is the only page that its not working for me. Not sure why, I don't even see a status code in the network tab

**Comment by xuelink** [comment]
> @xuelink Locally, the github stars works on all pages, blogs, privacy, terms and cookies and so on. But on the live website, cookies is the only page that its not working for me. Not sure why, I don't even see a status code in the network tab

@DH-555 What do you think about that ? Maybe its related with Cloudflare :/ i have no idea

**Comment by DH-555** [comment]
No idea, will check by the weekend, but it seems that it's not being made any request in that specific page in order to fetch the data from GitHub.
